controllers/webhook/runcompletion_feed.go

```diff
@@ -0,0 +1,111 @@
+package webhook
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	config "github.com/sky-uk/kfp-operator/apis/config/v1alpha5"
+	"github.com/sky-uk/kfp-operator/apis/pipelines"
+	"io"
+	"net/http"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+type EventData struct {
+	// Header is the http request header
+	Header http.Header `json:"header"`
+	// Body is http request body
+	Body json.RawMessage `json:"body"`
```

Comment:
Wonder if these comments are necessary and we could just name the fields better? like `HttpRequestHeader` and `HttpRequestBody` ?

Reply:
Agree, comments are pointless but think the names should stay generic

Reply:
[d626eca](https://github.com/sky-uk/kfp-operator/pull/368/commits/d626ecacae21d54eb78ccd9c1dcb1075cbef2215)